README.rst

```diff
@@ -65,21 +65,24 @@ command invocations:
 
 .. code-block:: text
 
-   $ usbsdmux -h
-   usage: usbsdmux [-h] SG {get,dut,client,host,off}
-
-   positional arguments:
-     SG                    /dev/sg* to use
-     {get,dut,client,host,off}
-			   Action:
-			   get - return selected mode
-			   dut - set to dut mode
-			   client - set to dut mode (alias for dut)
-			   host - set to host mode
-			   off - set to off mode
-
-   optional arguments:
-     -h, --help            show this help message and exit
+    $ usbsdmux -h
```

Comment:
This changes the indention from the conventional three spaces to four spaces.

Reply:
> … conventional three spaces …

Tehe, the reStructuredText format never fails to surprise me.